crates/holochain/src/test_utils.rs

```diff
@@ -699,18 +699,21 @@ pub async fn get_integrated_ops<Db: ReadAccess<DbKindDht>>(db: &Db) -> Vec<DhtOp
 }
 
 /// Helper for displaying agent infos stored on a conductor
-pub async fn display_agent_infos(_conductor: &ConductorHandle) {
-    todo!()
-    /*
-    for cell_id in conductor.running_cell_ids() {
-        let space = cell_id.dna_hash();
-        let db = conductor.get_p2p_db(space);
-        let info = p2p_agent_store::dump_state(db.into(), Some(cell_id))
+pub async fn display_agent_infos(conductor: &ConductorHandle) {
```

Comment:
This isn't actually used. It's a useful utility I think, but we really ought to use it somewhere to check it... There's lot of things like that in sweettest, so I thought I'd restore it for now and it can get looked at again if we ever get to extracting sweettest.